cmd/minikube/cmd/start.go

```diff
@@ -197,6 +201,11 @@ func initDriverFlags() {
 
 	// hyperv
 	startCmd.Flags().String(hypervVirtualSwitch, "", "The hyperv virtual switch name. Defaults to first found. (only supported with HyperV driver)")
+
+	// kic
+	startCmd.Flags().String(kicImg, "", "override the kic image. (Only to be used with on with kic driver) ")
+	startCmd.Flags().String(ociClient, "docker", "the oci client to be used. (Only to be used with on with kic driver)")
```

Comment:
To avoid introducing an additional level of confusing concepts, I think this should use  `--driver=docker` (and rename --vm-driver to --driver)